libs/assets/src/lib/asset-details-table.tsx

```diff
@@ -257,3 +266,16 @@ export const AssetDetailsTable = ({
     </KeyValueTable>
   );
 };
+
+// Separate component for the link as otherwise eslint will complain
+// about useEnvironment being used in a component
+// named with a lowercase 'value'
+const ContractAddressLink = ({ address }: { address: string }) => {
+  const { ETHERSCAN_URL } = useEnvironment();
+  const href = `${ETHERSCAN_URL}/address/${address}`;
```

Comment:
Some time ago I've put all links related logic into one convenient place, please use `useEtherscanLink` from `@vegaprotocol/environment`

```suggestion
  const etherscanLink = useEtherscanLink();
  const href = etherscanLink(address);
```